src/dlgAboutDialog.cpp

```diff
@@ -81,7 +81,7 @@ dlgAboutDialog::dlgAboutDialog(QWidget * parent) : QDialog(parent)
         }
 
         // Repeat for other text, but we know it will fit at given size
-        QString sourceCopyrightText = QChar( 169 ) % QString( " Heiko K" ) % QChar( 246 ) % QString( "hn 2008-" ) % QString(__DATE__).mid(7);
```

Comment:
Can we do this - with Heiko as the figure-head don't we need his approval to take his name off the front.

__figure-head__: _noun_, object at the pointy-end of a ship; usually the first part that gets damaged should the ship run into a solid obstruction...</small> :rofl: